docs/api-docs/management-apis/contextual-filters.md

```diff
@@ -0,0 +1,402 @@
+# Contextual Filters
+
+<div class="otp" id="no-index">
+
+### On this page
+- [View active storefront filters](#view-active-storefront-filters)
+- [View available storefront filters](#view-available-storefront-filters)
+- [View available category-level filters](#view-available-category-level-filters)
+- [View contextual filters](#view-contextual-filters)
+- [Configure contextual filters](#configure-contextual-filters)
+- [Related resources](#related-resources)
+
+</div>
+
+The [Settings API](https://developer.bigcommerce.com/api-reference/store-management/settings) allows you to manage settings and configurations for BigCommerce hosted stores and headless storefronts. 
+
+You can use the [Contextual Filters](insert-link-here) endpoint of the Settings API to programmatically configure a store's Product Filtering feature, also known as "faceted search".
+Additionally, the [Contextual Filters](insert-link-here) endpoint enables you to merchandise diverse product catalogs by creating a different faceted search configuration for each channel and product category on a store. With this feature, you can configure the product categories for various product lines with a different set of enabled filters, so that shoppers can filter by the most relevant criteria.
+
+This article demonstrates how to use [Contextual Filters](insert-link-here) to configure filters that appear on a particular category page.
+
+### Limitations
+
+* Currently, the API only supports the default channel with an ID of 1.
+* You can configure contextual filters for Category pages, but not for Brand or Search Results pages at this time.
+
+###  Prerequisites
+
+* [A BigCommerce store](https://support.bigcommerce.com/s/article/Starting-a-Bigcommerce-Trial).
+* API `access_token` with `content modify` scope.
```

Comment:
Is this the right scope? Settings APIS are supposed to be under the "Modify Store Information" scope.

Reply:
API `access_token` with `information modify` scope. - Is this correct?